FERMCHOP/FERMCHOP-IOC-01App/Db/fermchop.db

```diff
@@ -346,6 +365,13 @@ record(ai, "$(P)TEMPERATURE:MOTOR")
 	info(archive, "VAL")
 } 
 
+record(calc, "$(P)TEMPERATURE:RANGECHECK")
+{
+	field(INPA, "$(P)TEMPERATURE:MOTOR CP")
+	field(INPB, "$(P)TEMPERATURE:ELECTRONICS CP")
+	field(CALC, "A > 45 | B > 45")
```

Comment:
Can you add a comment as to where these values come from?

Reply:
The PV names are a bit long - also https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming suggests using  TEMP for temperature